src/BuildIt.Forms/BuildIt.Forms.Controls/Enums.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace BuildIt.Forms.Controls
+{
+    public enum CameraFocusMode
+    {
+        Auto,
```

Comment:
Can we add comments around this to indicate what the difference is between Auto and Continous. Also, should include None or Default as the first value, so that there is a differentiation between no value (ie None) and Auto (the first actual value)

Reply:
the only problem with adding the Default/None, particularly in this case is that given there's a method to return the supported focus modes is does that if there are no supported focus modes it could mean that the method should return a collection with the None value in there, which doesn't seem right. Autofocus appears to be the default for all platforms and would fit with the guidelines of using the most frequently used member as per this [link](https://msdn.microsoft.com/en-us/library/ms182149.aspx). Can still add None if you want though

Reply:
No it doesn't imply that - the None just makes it easier to deal with enum variables. The collection of focus modes doesn't need to include None.